v21.2/deploy-lambda-function.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Create and Deploy an AWS Lambda Function on CockroachDB
+summary: Learn how to use AWS Lambda and CockroachDB Serverless.
+toc: true
+twitter: false
+referral_id: docs_lambda_python
+---
+
+This tutorial shows you how to create an [AWS Lambda](https://aws.amazon.com/lambda) function that communicates with a {{ site.data.products.serverless }} cluster.
+
+The example function used for this tutorial is written in Python. The function uses the [Psycopg](https://www.psycopg.org/) PostgreSQL adapter to connect to CockroachDB.
+
+## Prerequisites
+
+Before starting the tutorial, do the following:
+
+1. Create a [{{ site.data.products.db }}](https://cockroachlabs.cloud/signup?referralId={{page.referral_id}}) account.
+
+1. Create an [AWS](https://aws.amazon.com/) account.
+
+1. Install the [AWS CLI](https://aws.amazon.com/cli/).
+
+## Step 1. Create a {{ site.data.products.serverless }} cluster
+
+{% include cockroachcloud/quickstart/create-a-free-cluster.md %}
+
+<a name="connection-string"></a>
+
+After the cluster is created, the **Connection info** window appears. Click the **Connection string** tab and copy the connection string to a secure location. You will use this connection string to connect to CockroachDB later in the tutorial.
+
+{{site.data.alerts.callout_info}}
+The connection string is pre-populated with your username, cluster name, and other details, including your password. Your password, in particular, will be provided only once. Save it in a secure place (we recommend a password manager) to connect to your cluster in the future. If you forget your password, you can reset it by going to the [**SQL Users** page](../cockroachcloud/user-authorization.html).
+{{site.data.alerts.end}}
+
+## Step 2. Get the sample code
+
+Open a terminal window and copy the sample code's GitHub repo:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+$ git clone https://github.com/cockroachlabs/examples-aws-lambda/
+~~~
+
+The function's code is available under the `examples-aws-lambda/python` directory:
+
+~~~ shell
+.
+├── README.md
+├── deployment-package.zip  ## Lambda deployment package
+├── init_db.py              ## Lambda function source code
+├── package                 ## Psycopg dependencies
+├── requirements.txt        ## List of Python requirements
+└── root.crt                ## CA cert
```

Comment:
The goal is eventually to remove the hard-dependency of `root.crt` in connecting to Serverless clusters, fwiw.

Reply:
That day cannot come soon enough.